### PR TITLE
:wrench: [#2228] Prepare settings for i18n

### PR DESCRIPTION
**Pull request by stevenbal**
Fixes: #2228 

Changes:
* Add settings for i18n

### CODE REVIEW

File: src/openforms/conf/base.py [hunk 1]
```diff
@@ -47,6 +47,14 @@
     ("nl", _("Dutch")),
 ]
 
+LANGUAGE_COOKIE_HTTPONLY = True
+
+LANGUAGE_COOKIE_NAME = "openforms_language"
```
Comment (stevenbal) [inline src/openforms/conf/base.py:52]: Not sure if we want this to be configurable

Comment (sergei-maertens) [reply src/openforms/conf/base.py:52]: nope, fixed value is fine (session cookie name is also fixed)

File: src/openforms/conf/base.py [hunk 2]
```diff
@@ -47,6 +47,14 @@
...
+
+LANGUAGE_COOKIE_SAMESITE = "Lax"
```
Comment (sergei-maertens) [inline src/openforms/conf/base.py:54]: this needs to be `"None"` for cross-origin use, see the similar CSRF & Session cookie setings

Comment (stevenbal) [reply src/openforms/conf/base.py]: Ah okay, so whenever forms are embedded, you would want this to be "None"

Comment (sergei-maertens) [reply src/openforms/conf/base.py]: yep!

File: src/openforms/conf/base.py [hunk 3]
```diff
@@ -47,6 +47,16 @@
...
 
+LANGUAGE_COOKIE_HTTPONLY = True
+
+LANGUAGE_COOKIE_NAME = "openforms_language"
+
+LANGUAGE_COOKIE_SAMESITE = config(
+    "LANGUAGE_COOKIE_SAMESITE", default="None" if IS_HTTPS else "Lax"
+)
+
+LANGUAGE_COOKIE_SECURE = IS_HTTPS
```
Comment (sergei-maertens) [inline src/openforms/conf/base.py:58]: final remark - can you remove the blank lines between the language settings to be consistent with the rest of the settings file? :grimacing:

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**